src/main/main.c

```diff
@@ -391,6 +391,20 @@ void init(void)
     adcInit(&adc_params);
 #endif
 
+    /* Extra 500ms delay prior to initialising hardware if board is cold-booting */
```

Comment:
Could we put this under a `#if defined(GPS) || defined(MAG)` , so that startup in not unnecessarily delayed.

I imagine there are a few people who use iNav for plain old line of sight flying...